### PR TITLE
descending order based on date

### PR DESCRIPTION
**Pull request by agramfort**
for some reason this does not work

help welcome @tanglef 

basically I want to see the latest results appear first

### CODE REVIEW

File: benchopt/plotting/generate_html.py [hunk 1]
```diff
@@ -495,7 +495,7 @@ def plot_benchmark_html_all(patterns=(), benchmarks=(), root=None,
         fnames = []
         for p in patterns:
             fnames += (benchmark / 'outputs').glob(f"{p}.csv")
-        fnames = sorted(set(fnames))
+        fnames = sorted(set(fnames))[::-1]
```
Comment (tanglef) [inline benchopt/plotting/generate_html.py:498]: The elements are not sorted here but in the JS table because of jquery/aria anyway, so this can be removed

**Review by tanglef — CHANGES_REQUESTED** [review]
To do that, we actually need to touch the JS in `benchmark.mako.html`.
I will do a PR on your branch with the changes needed

**Comment by agramfort** [comment]
can you have a look?

we try to push these days the final touches to benchopt to push a new
release
that should be the first stable release.

thx !

>

**Comment by tanglef** [comment]
I can't push directly on your branch so I made a PR (https://github.com/agramfort/benchOpt/pull/1) on your forked version of the repo. If that is ok for you, you can merge it on your branch and it will be all good here.

**Comment by tanglef** [comment]
And the CI seem to be good today :)

**Comment by agramfort** [comment]
works like a charm @tanglef  thx heaps !

**Pull request merged**